<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySalary</class>
 <widget class="QMainWindow" name="MySalary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySalary</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="welcome">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="1">
           <widget class="QLineEdit" name="username">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>UserName</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="tip">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>You must login to continue</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_username">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Username:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="loginButton">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_password">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="slogan">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Welcome to MySalary!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="password">
            <property name="font">
             <font>
              <family>Microsoft YaHei</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="locale">
             <locale language="Chinese" country="China"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="content">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>6</number>
          </property>
          <widget class="QWidget" name="salary">
           <attribute name="title">
            <string>salary</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTableView" name="salaryView"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="bonus">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>bonus</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <widget class="QTableView" name="bonusView">
              <property name="styleSheet">
               <string notr="true">width: 100%</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="info">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>info</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0" rowspan="2">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_staffID">
                    <property name="text">
                     <string>staffID:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="staffID">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_name">
                    <property name="text">
                     <string>name:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="name">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label_grade">
                    <property name="text">
                     <string>grade:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="gradeName">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_loginName">
                    <property name="text">
                     <string>loginName:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="new_loginName"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_password_2">
                    <property name="text">
                     <string>password:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="new_password">
                    <property name="echoMode">
                     <enum>QLineEdit::Password</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_info">
                  <property name="text">
                   <string>info:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="new_info"/>
                </item>
               </layout>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="infoCommitButton">
                <property name="text">
                 <string>Commit Info</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="admin_grade">
           <attribute name="title">
            <string>admin_grade</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0" rowspan="6">
             <widget class="QTableView" name="gradeView"/>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="grade_CommitButton">
              <property name="text">
               <string>Commit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="grade_CancelButton">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="grade_AddButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="grade_DeleteButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QRadioButton" name="grade_ASCButton">
              <property name="text">
               <string>ASC</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QRadioButton" name="grade_DESCButton">
              <property name="text">
               <string>DESC</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="admin_staff">
           <attribute name="title">
            <string>admin_staff</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0" rowspan="5" colspan="2">
             <widget class="QTableView" name="staffView"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="staff_CommitButton">
              <property name="text">
               <string>Commit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="staff_CancelButton">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="staff_AddButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="staff_DeleteButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QRadioButton" name="staff_ASCButton">
              <property name="text">
               <string>ASC</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QComboBox" name="filter_comboBox"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="filter">
              <property name="placeholderText">
               <string>Keyword</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QRadioButton" name="staff_DESCButton">
              <property name="text">
               <string>DESC</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="admin_bonus">
           <attribute name="title">
            <string>admin_bonus</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0" rowspan="5" colspan="2">
             <widget class="QTableView" name="admin_bonus_View"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="bonus_CommitButton">
              <property name="text">
               <string>Commit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="bonus_CancelButton">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="bonus_AddButton">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QPushButton" name="bonus_DeleteButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QRadioButton" name="bonus_ASCButton">
              <property name="text">
               <string>ASC</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QComboBox" name="bonus_filter_comboBox"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="bonus_filter">
              <property name="placeholderText">
               <string>Keyword</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QRadioButton" name="bonus_DESCButton">
              <property name="text">
               <string>DESC</string>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>bonus_AddButton</zorder>
           <zorder>bonus_CommitButton</zorder>
           <zorder>bonus_ASCButton</zorder>
           <zorder>bonus_DESCButton</zorder>
           <zorder>bonus_DeleteButton</zorder>
           <zorder>bonus_filter_comboBox</zorder>
           <zorder>bonus_filter</zorder>
           <zorder>admin_bonus_View</zorder>
           <zorder>bonus_CancelButton</zorder>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>admin_salary</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0" colspan="2">
             <widget class="QTableView" name="admin_salaryView"/>
            </item>
            <item row="1" column="0">
             <widget class="QDateEdit" name="admin_salary_dateEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="admin_payButton">
              <property name="text">
               <string>Pay!</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
